api/src/wfl/module/wgs.clj

```diff
@@ -112,6 +112,12 @@
     (all/throw-when-output-exists-already! output)
     (util/prefix-keys inputs :ExternalWholeGenomeReprocessing)))
 
+(defn make-labels
+  "Return labels for wgs pipeline from OTHER-LABELS."
+  [other-labels]
+  (merge cromwell-label-map
+         other-labels))
```

Comment:
It looks like other-labels gets passed around a lot.
Maybe this can be factored to be more direct?
Start accreting labels where the UUID is generated?